magma/bits.py

```diff
@@ -532,7 +537,7 @@ def simulate(self, value_store, state_store):
 
     @lru_cache(maxsize=None)
     def Define(width):
-        return m.circuit.DeclareCoreirCircuit(
+        return DeclareCoreirCircuit(
```

Comment:
Another deprecation warning point

Reply:
Definitely; will put up a follow up PR to get rid of more of these.